packages/KSlideout/KSlideout.vue

```diff
@@ -87,7 +87,7 @@ export default {
     right: 0;
     bottom: 0;
     left: 0;
-    background: var(--tblack-45);
+    background: var(--tblack-45, color(tback-45));
```

Comment:
tblack, not tback

Reply:
Epic typo >_<

In fact, `color()` is not working, let go back to `hardcoded` style.